titiler/errors.py

```diff
@@ -1,4 +1,15 @@
 """Titiler error classes."""
+from typing import Callable, Dict, Type
+
+from cogeo_mosaic.errors import MosaicAuthError, MosaicError, MosaicNotFoundError
+from rasterio.errors import RasterioError, RasterioIOError
+from rio_tiler.errors import InvalidBandName, RioTilerError, TileOutsideBounds
```

Comment:
Maybe we can add basic rasterio errors like https://github.com/mapbox/rasterio/blob/master/rasterio/errors.py#L40